dts/bindings/usb/nxp,kinetis-usbd.yaml

```diff
@@ -22,3 +22,8 @@ properties:
 
     interrupts:
       category: required
+
+    clocks:
+      type: array
```

Comment:
Now that clocks is in base.yaml, we just need 'category: required' here the type, and description aren't needed.

Reply:
Done